LondonBritishLibrary/orient/BLorient765.xml

```diff
@@ -104,13 +73,137 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             <citedRange unit="page">172b-172b</citedRange>
                                         </bibl>
                                     </listBibl>
-                                    
+
                                 </source>
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+<msPart xml:id="p1">
+<msIdentifier></msIdentifier>
+<msContents>
+
+<msItem xml:id="p1_i1">
+<locus target="#3"/>
```

Comment:
3? the text of the second part start on fol. 3 as well...

Reply:
It is correct here, but see below

Reply:
So I'm leaving it as it is.